pkg/common/security/usergroup_test.go

```diff
@@ -231,12 +336,14 @@ func TestConvertUGI(t *testing.T) {
 	if err != nil {
 		t.Errorf("unknown user with groups, convert should not have failed: %v", err)
 	}
+	testCache.lock.Lock()
 	if ug.User != "unknown2" || len(ug.Groups) != 1 || ug.resolved == 0 || ug.failed {
 		t.Fatalf("User 'unknown2' not resolved correctly: %v", ug)
 	}
 	if ug.Groups[0] != group {
 		t.Errorf("groups not initialised correctly on convert: expected '%s' got '%s'", group, ug.Groups[0])
 	}
+	testCache.lock.Unlock()
```

Comment:
This lock/unlock pair can be removed. `ConvertUGI()` returns a copy and internally uses a mutex to access the map.